client/src/components/common/SmallTileDetails.tsx

```diff
@@ -24,6 +24,14 @@ export const SmallTileDetails: React.FC<{
             text-overflow: ellipsis;
             white-space: wrap;
           }
+
+          @media screen and (max-width: ${bp.small}px) {
+            > div {
```

Comment:
Shouldn't you just define this on the div on the next line?

Reply:
No cause it's for all 3

Reply:
Then it should be styled component :P

Reply:
Cleaned up !